_posts/2021-05-30-cloudnative-bot.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Using OpenFaas to run a fun twitter bot"
+description: "Using OpenFaas to run a fun twitter bot"
```

Comment:
The description can be 2-3 times longer and should be like the title, but a little more verbose.